server/neptune/temporalworker/server.go

```diff
@@ -223,6 +231,16 @@ func (s Server) Start() error {
 		s.Logger.InfoContext(ctx, "Shutting down terraform worker, resource clean up may still be occurring in the background")
 	}()
 
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+
+		prRevisionWorker := s.buildPRRevisionWorker()
+		if err := prRevisionWorker.Run(worker.InterruptCh()); err != nil {
+			log.Fatalln("unable to start pr revision worker", err)
+		}
+	}()
+
```

Comment:
I thought we were going to discuss this part, whether to have it as a standalone process or not.  We can start off with this since it's already done but it might not be the most ideal to keep in the long run.

Reply:
From our discussion in the rebase flow sync meeting, I thought we wanted to go this route for v1. I've added a TODO comment to investigate spinning up a new worker for the PR Revision worker process.